front/src/common/ScenarioExplorer/ScenarioExplorerModal.tsx

```diff
@@ -65,8 +66,11 @@ export default function ScenarioExplorerModal({
         .then(({ results }) => setStudiesList(results))
         .catch((error) => console.error(error));
     }
-    setStudyID(undefined);
-    setScenariosList(undefined);
+    if (projectID !== globalProjectId || studyID !== globalStudyId) {
+      // not initialization
```

Comment:
not initialized, or no initialization ?

Reply:
It stands for "it's not initialization", meaning that the component has already been initialized :+1: